pkg/cadvisor/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 as build
+
+RUN apk add --no-cache bash go git g++ linux-headers make
+
+ENV GOPATH=/go PATH=$PATH:/go/bin
+ENV GITREPO=github.com/google/cadvisor
+ENV COMMIT=release-v0.28
+
+ADD /static.patch /tmp/
+
+RUN go get -d ${GITREPO} \
+    && cd /go/src/${GITREPO} \
+    && git checkout ${COMMIT} \
+    && patch -p1 build/build.sh </tmp/static.patch \
+    && make build \
+    && mv cadvisor /usr/bin/
+
+# -----------------------------------------------------------
```

Comment:
similar to my comment on https://github.com/linuxkit/linuxkit/pull/2754 I prefer to not having these dividers. Minor nit, though

Reply:
of course